snapcraft/tests/test_sources.py

```diff
@@ -321,6 +321,22 @@ def test_init_with_source_branch_and_tag_raises_exception(self):
         self.assertEqual(raised.exception.message, expected_message)
 
 
+class TestSubversion(SourceTestCase):
+
+    def test_pull(self):
+        svn = sources.Subversion('svn://my-source', 'source_dir')
+        svn.pull()
+        self.mock_run.assert_called_once_with(
+            ['svn', 'checkout', 'svn://my-source', 'source-dir'])
+
+    def test_pull_existing(self):
+        self.mock_path_exists.return_value = True
+        svn = sources.Subversion('svn://my-source', 'source_dir')
+        svn.pull()
+        self.mock_run.assert_called_once_with(
+            ['svn', 'update', 'source-dir'])
```

Comment:
@elopio This line is causing something to fail, I think the unit tests? https://travis-ci.org/ubuntu-core/snapcraft/jobs/137352934

What do I need to correct to make this go away? Do I need to remove `'svn://my-source'` from line 334?